app/helper.go

```diff
@@ -0,0 +1,30 @@
+package app
+
+import (
+	"context"
+	"errors"
+
+	"google.golang.org/grpc/metadata"
+)
+
+type helper struct {
+	authenticatedUserHeaderKey string
+}
+
+func NewHelper(authenticatedUserHeaderKey string) *helper {
+	return &helper{authenticatedUserHeaderKey}
+}
+
+func (h helper) GetAuthenticatedUser(ctx context.Context) (string, error) {
```

Comment:
Is it possible to move this to the `user` domain context instead of having a generic `helper` ?

Reply:
this is actually no longer being used, will remove this